collector/process.go

```diff
@@ -2,11 +2,78 @@
 
 package collector
```

Comment:
Given the amount of code and requirement for extra libraries, I'd be inclined to suggest we put the per-process GPU metrics in a separate collector. What do you think?

Reply:
Hmm, well, as for the extra libs, this only uses DXGI.lib, which is present in Windows/System32. The data comes from perflib, the extra code is just to match the GPU luids with the corresponding processes, so since I perceive the process collector as something that takes data from perflib, makes sense for me that this stuff lives here, but again we can talk about it.